Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/connectmgr/persistance/dao/ConnectionManagerDAOBase.java

```diff
@@ -63,13 +65,19 @@ protected void saveBusinessDetail(BusinessDetail BusinessDetail, File file) {
             synchronized (file) {
                 JAXBContext context = JAXBContext.newInstance(BusinessDetail.class);
                 ObjectFactory factory = new ObjectFactory();
-                Marshaller marshaller = context.createMarshaller();
-                marshaller.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, Boolean.TRUE);
+                DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();        
```

Comment:
Please verify the line number on the ticket.  The issue is on method loadBusinessDetail (line 54) not on saveBusinessDetail.  In additional to that, I don't see the benefit on new block that you introduce.  They are unused.